build-headless-shell.sh

```diff
@@ -181,9 +181,10 @@ if [ "$SYNC" -eq "1" ]; then
 #  import(\"$SRC/icecc-chromium/icecc.gni\") # icecc parameters (removed)
   echo "import(\"//build/args/headless.gn\")
   is_debug=false
+  is_official_build=true
   symbol_level=0
-  enable_nacl=false
```

Comment:
Move `enable_nacl=false` after `blink_symbol_level=0` so that `symbol_level=0` and `blink_symbol_level=0` will stick together. This change is just a re-format.